package.json

```diff
@@ -9,7 +9,8 @@
   },
   "scripts": {
     "prepublish": "gulp",
-    "test": "mocha --recursive --require intelli-espower-loader --require source-map-support/register --reporter spec"
+    "test": "mocha --recursive --require intelli-espower-loader --require source-map-support/register --reporter spec",
+    "postinstall": "node_modules/.bin/babel --recursive  --stage 0 -d node_modules/protodef/dist/ node_modules/protodef/src/"
```

Comment:
because of the npm bug about gulp + git dependencies. I'll remove it once protodef new version is published.